kthresher.systemd.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Purge unused kernels
+Documentation=man:kthresher(8)
+
+[Service]
+Type=oneshot
+# by default, do nothing but provide information
+ExecStart=/usr/bin/kthresher --dry-run
```

Comment:
It's OK to *not* use `--dry-run` the [default config](https://github.com/rackerlabs/kthresher/blob/master/kthresher.conf) does not purge any package unless it is defined.